tensorflow/compiler/tf2tensorrt/convert/ops/variable_ops.cc

```diff
@@ -217,6 +217,194 @@ class ConvertVariableV2 : public OpConverterBase<ConvertVariableV2> {
 REGISTER_DEFAULT_TRT_OP_CONVERTER(MakeConverterFunction<ConvertVariableV2>(),
                                   {"VariableV2"});
 
+class ConvertReadVariableOp : public OpConverterBase<ConvertReadVariableOp> {
+ public:
+  ConvertReadVariableOp(OpConverterParams* params)
+      : OpConverterBase<ConvertReadVariableOp>(params) {}
+
+  struct ReadVariableOpAttributes {
+    TensorShapeProto shape_proto;
+    TensorShape shape;
+    string name;
+    DataType dtype;
+  };
+
+  static constexpr std::array<InputArgSpec, 1> InputSpec() {
+    return {InputArgSpec::Create("resource", TrtInputArg::kResource)};
+  }
+
+  static constexpr std::array<DataType, 2> AllowedDataTypes() {
+    return {DataType::DT_FLOAT, DataType::DT_HALF};
```

Comment:
When I allowed `INT32` for `VariableV2` and added a test case, I was hitting an error that there was no registered kernel for `INT32`, and I assumed the same for `ReadVariableOp` (after all, `INT32` is associated with shapes, it would typically be a `Const`), but I just checked and I'm not hitting that error with `ReadVariableOp`.

However, I'm seeing other errors, which I think are due to TF running `INT32` ops on the host (some send/receive ops in my tests are complaining that the rendezvous isn't set up).

I'll see if efforts should be made now to make the converter work with `INT32`.